### PR TITLE
feat(cli): use built-in scanner for configuration audits

### PR DESCRIPTION
**Pull request by danielpacak**
Signed-off-by: Daniel Pacak <pacak.daniel@gmail.com>

### CODE REVIEW

File: pkg/configauditreport/controller.go [hunk 1]
```diff
@@ -260,10 +260,10 @@ func (r *ResourceController) policies(ctx context.Context) (*policy.Policies, er
 
 	err := r.Client.Get(ctx, client.ObjectKey{
 		Namespace: r.Config.Namespace,
-		Name:      "starboard-policies-config",
+		Name:      starboard.PoliciesConfigMapName,
 	}, cm)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed getting policies from configmap: %q: %w", r.Config.Namespace+"/"+starboard.PoliciesConfigMapName, err)
```
Comment (chen-keinan) [inline pkg/configauditreport/controller.go:266]: with format , can also format param : 
```
fmt.Errorf("failed getting policies from configmap: %s/%s: %w", r.Config.Namespace ,starboard.PoliciesConfigMapName,err)
```eady using format , can also format param : 
```
fmt.Errorf("failed getting policies from configmap: %s/%s: %w", r.Config.Namespace ,starboard.PoliciesConfigMapName,err)
```

File: pkg/configauditreport/scanner.go [hunk 2]
```diff
@@ -4,149 +4,116 @@ import (
...
-		}
+func (s *Scanner) policies(ctx context.Context) (*policy.Policies, error) {
+	cm := &corev1.ConfigMap{}
+
+	err := s.client.Get(ctx, client.ObjectKey{
+		Namespace: starboard.NamespaceName,
+		Name:      starboard.PoliciesConfigMapName,
+	}, cm)
+	if err != nil {
+		return nil, fmt.Errorf("failed getting policies from configmap: %q: %w", starboard.NamespaceName+"/"+starboard.PoliciesConfigMapName, err)
```
Comment (chen-keinan) [inline pkg/configauditreport/scanner.go:116]: withformat , can also format param : 
```
fmt.Errorf("failed getting policies from configmap: %s/%s: %w", r.Config.Namespace ,starboard.PoliciesConfigMapName,err)
```

File: pkg/configauditreport/scanner.go [hunk 3]
```diff
@@ -4,149 +4,116 @@ import (
...
+	data := v1alpha1.ConfigAuditReportData{
+		Scanner: v1alpha1.Scanner{
+			Name:    "Starboard",
+			Vendor:  "Aqua Security",
+			Version: s.buildInfo.Version,
+		},
+		Summary: v1alpha1.ConfigAuditSummaryFromChecks(checks),
+		Checks:  checks,
+
+		PodChecks:       checks,
```
[the remark this answers is not in the document]
Comment (danielpacak) [reply pkg/configauditreport/scanner.go:88]: Yes. We keep duplicates until PodChecks and ContainerChecks properties are removed.

[the remark this answers is not in the document]
Comment (danielpacak) [reply pkg/configauditreport/scanner.go:88]: I've created #1029 to keep track of it.

**Review by chen-keinan — APPROVED** [review]

**Pull request merged**